coderedcms/admin_urls.py

```diff
@@ -1,10 +1,12 @@
 from django.conf.urls import include, url
+from wagtailimportexport import urls as wagtailimportexport_urls
 from wagtail.admin import urls as wagtailadmin_urls
-from coderedcms.views import clear_cache
-from coderedcms.settings import cr_settings
+from coderedcms.views import clear_cache, import_pages_from_csv_file
 
 
 urlpatterns = [
     url(r'^codered/clearcache$', clear_cache, name="clear_cache"),
+    url(r'^import-export/import_from_csv/$', import_pages_from_csv_file, name="import_from_csv"),
```

Comment:
This follows the same url scheme that `wagtailimportexport` uses.  Is prefixing with codered still preferable?

Reply:
Well, if the `import_from_csv` is our own view, then I would prefix with codered. If it is their view, then keep as-is. We might want to try and contribute these changes back to the import-export pip package in the future if it is generic enough.

Reply:
It's our own view that copies much of what wagtail does with their importing.  I had to change a few things to match the use case of our csv's.  I don't think it would be generic enough for that package.  Their idea is that you're doing whole/partial tree dumps/uploads.  Not giving it a csv and telling it you want these entries as pages.  I can see the argument for using our url from that perspective, I just wasn't sure if it was correct.